cfgov/searchgov/forms.py

```diff
@@ -1,14 +1,12 @@
 from django import forms
 
-from core.utils import make_safe
-
 
 class SearchForm(forms.Form):
     q = forms.CharField(strip=True)
     page = forms.IntegerField(required=False)
 
     def clean_q(self):
-        return make_safe(self.cleaned_data["q"])
+        return self.cleaned_data["q"]
```

Comment:
You can delete this whole method since you're no longer modifying `q`.

Reply:
Done in 5f0e22cd75b31058a562cd747d54af99be23f9c7